lib3h_protocol/src/error/mod.rs

```diff
@@ -0,0 +1,41 @@
+//! Lib3h API Error Enum
+
+pub mod sodium_error;
+
+use sodium_error::SodiumError;
+
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub enum Lib3hError {
```

Comment:
So, from what I can read, `lib3h_protocol` is responsible for the `error` handling of the whole crate ? Not the root `lib3h` ?

Reply:
Discussed IRL. I'm going to move this into a new `holochain_crypto_api` crate :+1: